docs/_ext/autorobusta.py

```diff
@@ -244,37 +225,29 @@ def run(self) -> List[Node]:
         action_definition = Action(obj)
         return self.__generate_rst(action_definition, recommended_trigger)
 
-    def __generate_rst(
-        self, action_definition: Action, recommended_trigger: Optional[str]
-    ):
+    def __generate_rst(self, action_definition: Action, recommended_trigger: Optional[str]):
         node = nodes.section()
         node.document = self.state.document
 
         trigger_params = json.loads(self.options.get("trigger-params", "{}"))
 
-        example_yaml = generator.generate_example_config(
-            action_definition.func, recommended_trigger, trigger_params
-        )
+        example_yaml = generator.generate_example_config(action_definition.func, recommended_trigger, trigger_params)
         params_cls = action_definition.params_type
         params_cls_path = ""
         if params_cls is not None:
             params_cls_path = f"{params_cls.__module__}.{params_cls.__name__}"
 
-        code = self.__get_source_code(action_definition.func)
+        # code = self.__get_source_code(action_definition.func)
         description = self.__get_description(action_definition)
-        triggers = self.__get_triggers(
-            generator.get_supported_triggers(action_definition), recommended_trigger
-        )
+        triggers = self.__get_triggers(generator.get_supported_triggers(action_definition), recommended_trigger)
         cli_trigger = generator.get_manual_trigger_cmd(action_definition)
 
-        indented_code = "\n".join(" " * 32 + l for l in code)
-        indented_description = "\n".join(" " * 28 + l for l in description.split("\n"))
-        indented_example = "\n".join(" " * 32 + l for l in example_yaml.split("\n"))
-        indented_triggers = "\n".join(" " * 24 + l for l in triggers)
+        # indented_code = "\n".join(" " * 32 + line for line in code)
```

Comment:
this too?

Reply:
Same - it wasn't used
I think it was just left there after some changes